File: services/teams-service/Models/TeamVacancy.cs
namespace TeamsService.Models
{
    public class TeamVacancy : BaseModel
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public int TeamId { get; set; }
        public Team Team { get; set; }

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: services/main-service/app/Models/TeamVacancy.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class TeamVacancy extends Model
{
    use HasFactory;

    protected $fillable = [
        'title',
        'description',
        'team_id'
    ];

    /**
     * @return BelongsTo
     */
    public function team(): BelongsTo
    {
        return $this->belongsTo(Team::class);
    }
}
